specification.md

```diff
@@ -1910,26 +1910,25 @@ we can use this expression in the workflow "version" parameter:
 
 ```json
 {
-   "id": "MySampleWorkflow",
-   "name": "Sample Workflow",
+   "name": "MySampleWorkflow",
```

Comment:
Name is invalid here, too

Reply:
Yes, Im currently searching for mislaligments on specificatin.md